<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2D Cad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="CanvasContainer" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionLine"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionCircle"/>
    <addaction name="separator"/>
    <addaction name="actionSelect"/>
    <addaction name="actionResize"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionClear_Canvas"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionLine"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionCircle"/>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionResize"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionClear_Canvas"/>
  </widget>
  <action name="actionSelect">
   <property name="icon">
    <iconset resource="Resouces.qrc">
     <normaloff>:/rec/img/Select.png</normaloff>:/rec/img/Select.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset resource="Resouces.qrc">
     <normaloff>:/rec/img/line.png</normaloff>:/rec/img/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="Resouces.qrc">
     <normaloff>:/rec/img/rectangle.png</normaloff>:/rec/img/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="icon">
    <iconset resource="Resouces.qrc">
     <normaloff>:/rec/img/circle.png</normaloff>:/rec/img/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="icon">
    <iconset resource="Resouces.qrc">
     <normaloff>:/rec/img/resize.png</normaloff>:/rec/img/resize.png</iconset>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
  </action>
  <action name="actionClear_Canvas">
   <property name="icon">
    <iconset resource="Resouces.qrc">
     <normaloff>:/rec/img/remove-icon.png</normaloff>:/rec/img/remove-icon.png</iconset>
   </property>
   <property name="text">
    <string>Clear Canvas</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resouces.qrc">
     <normaloff>:/rec/img/save.png</normaloff>:/rec/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="Resouces.qrc">
     <normaloff>:/rec/img/save as.png</normaloff>:/rec/img/save as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resouces.qrc">
     <normaloff>:/rec/img/folder.png</normaloff>:/rec/img/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resouces.qrc">
     <normaloff>:/rec/img/Undo.png</normaloff>:/rec/img/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resouces.qrc">
     <normaloff>:/rec/img/Redo.png</normaloff>:/rec/img/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resouces.qrc"/>
 </resources>
 <connections/>
</ui>
